test/mld.jl

```diff
@@ -150,43 +128,6 @@ case24_ub = run_mld(case24, ACPPowerModel, ipopt_solver)["objective"]
         @test isapprox(active_power_served(result), 0.5805121110548641; atol = 1e-1)
         @test all_gens_on(result)
     end
-    @testset "5-bus pti" begin
-        result = run_mld(case5_pti, DCPPowerModel, ipopt_solver)
-
-        #println(result["objective"])
-        @test result["termination_status"] == LOCALLY_SOLVED
-        @test isapprox(result["objective"], 124.1; atol = 1e0)
-        #println("active power: $(active_power_served(result))")
-        @test isapprox(active_power_served(result), 4.099999509047602; atol = 1e-2)
-        #println([bus["status"] for (i,bus) in result["solution"]["bus"]])
-        @test isapprox(bus_status(result, "1"), 1.0; atol = 1e-4)
-        @test isapprox(bus_status(result, "2"), 1.0; atol = 1e-4)
-        @test isapprox(bus_status(result, "3"), 1.0; atol = 1e-4)
-        @test isapprox(bus_status(result, "4"), 1.0; atol = 1e-4)
-        @test isapprox(bus_status(result, "10"), 1.0; atol = 1e-4)
-    end
-    @testset "5-bus pti weights" begin
-        data = PowerModels.parse_file("../test/data/case5.raw")
-        add_load_weights!(data)
-
-        result = run_mld(data, DCPPowerModel, ipopt_solver)
-
-        #println(result["objective"])
-        @test result["termination_status"] == LOCALLY_SOLVED
-        @test isapprox(result["objective"], 9311.0; atol = 1e0)
-        #println("active power: $(active_power_served(result))")
-        @test isapprox(active_power_served(result), 4.099998420607658; atol = 1e-2)
-        #println([bus["status"] for (i,bus) in result["solution"]["bus"]])
-
-        loads = result["solution"]["load"]
-
-        # load 1 is high priorty, load 2 is medium priorty
-        @test loads["1"]["status"] >= loads["2"]["status"]
-
-        # load 2 is medium priorty, loads 3/4 are low priorty
-        @test loads["2"]["status"] >= loads["3"]["status"]
-        @test loads["2"]["status"] >= loads["4"]["status"]
-    end
```

Comment:
Why are these tests being removed or where they moved some place else?

Reply:
tests have been added back in